.github/workflows/build_docs.yml

```diff
@@ -0,0 +1,122 @@
+name: Build Documentation
+on:
+  push:
+    branches:
+      - 'master'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+# Setup
+
+      - name: Checkout repo
+        uses: actions/checkout@v2
+        with:
+          submodules: recursive
+          fetch-depth: 0
+
+      - name: Set up JDK 8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+# Version info
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      # - name: Gradle Get version
+      #   uses: eskatos/gradle-command-action@v1
+      #   with:
+      #     arguments: getVersion -q
+      #     wrapper-cache-enabled: true
+      #     dependencies-cache-enabled: true
+
+      - name: Retrieve versions from ForgeDev
+        run: ./gradlew retrieveVersion -q
+
+      - name: Extract Forge version
+        id: forge_version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'build/forge_version.json'
+          prop_path: 'forge_version'
+
+      - name: Extract mapping channel
+        id: mapping_channel
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'build/forge_version.json'
+          prop_path: 'mapping_channel'
+
+      - name: Extract mapping version
+        id: mapping_version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'build/forge_version.json'
+          prop_path: 'mapping_version'
+
+      - name: Extract mc version
+        id: mc_main_version
+        uses: notiz-dev/github-action-json-property@release
+        with:
+          path: 'build/forge_version.json'
+          prop_path: 'mc_main_version'
+
+# Sources extract
+
+      - name: Load Cache
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./.java
+          key: ${{ steps.forge_version.outputs.prop }}-sources
+
+      # - name: Generate and copy sources using Gradle
+      #   if: steps.cache.outputs.cache-hit != 'true'
```

Comment:
Same here: as this is commented out, could this be removed? Especially since it looks to be a copy of the next step.